foreman-selinux/foreman-selinux.spec

```diff
@@ -51,6 +51,7 @@ BuildRequires:  /usr/bin/pod2man
 BuildArch:      noarch
 
 Requires:           selinux-policy >= %{selinux_policy_ver}
+Requires:           container-selinux >= 2.0
```

Comment:
1. Not sure this is necessary if the policy is optional.
1. container-selinux is not present in the base EL7 OS, so it probably shouldn't be a dependency.